### DISCUSSION TITLE
感谢提供gguf模型，Mixtral是一个传奇！准备使用 koboldcpp本地推理试试

### DISCUSSION THREAD

**Discussion by Dunkelicht** [ymcui/Chinese-Mixtral#14] [General]
终于看到中文Mixtral模型了，正在下载中，
之前看thebloke也推荐了：
https://github.com/LostRuins/koboldcpp 
做本地推理，单exe配置起来也容易，下载完毕后试试，先从q4_k试起来，koboldcpp 有自己的llama.cpp fork，需要多测试一下。

期待这个项目能火起来，加油！

**Comment by ymcui** [comment]
感谢支持。llama.cpp的GGUF格式目前已成为主流大模型推理格式。
根据llama.cpp官方统计有以下UI支持GGUF：

[iohub/collama](https://github.com/iohub/coLLaMA)
[janhq/jan](https://github.com/janhq/jan) (AGPL)
[nat/openplayground](https://github.com/nat/openplayground)
[Faraday](https://faraday.dev/) (proprietary)
[LMStudio](https://lmstudio.ai/) (proprietary)
[LostRuins/koboldcpp](https://github.com/LostRuins/koboldcpp) (AGPL)
[Mozilla-Ocho/llamafile](https://github.com/Mozilla-Ocho/llamafile)
[nomic-ai/gpt4all](https://github.com/nomic-ai/gpt4all)
[ollama/ollama](https://github.com/ollama/ollama)
[oobabooga/text-generation-webui](https://github.com/oobabooga/text-generation-webui) (AGPL)
[psugihara/FreeChat](https://github.com/psugihara/FreeChat)
[cztomsik/ava](https://github.com/cztomsik/ava) (MIT)
[ptsochantaris/emeltal](https://github.com/ptsochantaris/emeltal)
[pythops/tenere](https://github.com/pythops/tenere) (AGPL)
[semperai/amica](https://github.com/semperai/amica)
[withcatai/catai](https://github.com/withcatai/catai)
[Mobile-Artificial-Intelligence/maid](https://github.com/Mobile-Artificial-Intelligence/maid) (MIT)
